tests/debugger/test_debugger_exception_replay.py

```diff
@@ -270,8 +270,18 @@ def scrub_span(key, value):
                     return "<scrubbed>"
 
                 if key == "meta" and isinstance(value, dict):
+                    keys_to_remove = []
```

Comment:
Hi @bouwkast ,

When the test logic is modified, please do not use a prefix in PR title. Otherwise, you may take the risk of breaking of tracers CI (and face conplaints 😉 )

Reply:
Sorry you mean the `[.NET]` thing? I just figured since I was changing .NET logic I'd put that in the title 😬 

I don't really follow how I could break other tracers with this change (but I also haven't really worked in this area of system tests before so I may just not be aware)

Is the preference to not do that?

Reply:
Ohh I see what you mean
I updated a general code path for the tests :sorry: